README.md

```diff
@@ -30,11 +30,6 @@ $ npm install --save resin-request
 Documentation
 -------------
 
-**Note.** This module expects [`fetch`](https://developer.mozilla.org/en/docs/Web/API/Fetch_API)
```

Comment:
This note should be removed from all other dependent modules when they get the updated resin-request.